Maddori.Apple/Maddori.Apple/Global/UIComponent/View/AlertViewController.swift

```diff
@@ -218,10 +221,32 @@ final class AlertViewController: BaseViewController {
             print("Delete")
         case .join:
             // FIXME: - 팀 합류 api 연결
+            self.pushHomeViewController()
             print("Join")
```

Comment:
p1;

```suggestion
        case .join:
            // FIXME: - 팀 합류 api 연결
            self.pushHomeViewController()
            self.dispatchJoinTeam(type: .dispatchJoinTeam(teamId: teamId ?? 0, userId: UserDefaultStorage.userID.description))
```

요 부분이 추가되어야 할 것 같습니다 !

Reply:
[c0fbb7c](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/114/commits/c0fbb7cfa3f94e852ebcb453337f4fbd0060f1e1)

감사합니다!!